### ISSUE TITLE
Distributed Tracing doesn't connect with Browser spans

### ISSUE THREAD

**Issue by andrewhr** [newrelic/elixir_agent#213]
**Describe the bug**

We have a Browser application producing trace information, which hits both a Rails backend and a Elixir backend. Traces between Browser and Rails are connected, but no connections happen on the Elixir side. Sampling is not an issue here because Elixir dominates our NR traffic. I've followed the troubleshoot docs and checked that NewRelic headers hit the servers. OTOH, distributed tracing going from Elixir to Elixir or Elixir to Ruby works so far.

After some code diving into both agents, I've noticed that [Ruby's promotes span id to parent id][1]. I tried to simulate the request processing on a `iex` session using the following code:

```elixir
conn =
  %Plug.Conn{}
  |> Plug.Conn.put_req_header("newrelic", "<payload from a real browser session>")
NewRelic.DistributedTrace.Plug.call(conn, [])
NewRelic.DistributedTrace.Tracker.fetch(self())
```

The stored `NewRelic.DistributedTrace.Context` struct has new `trace_id`, `guid`, but no `parent_id` neither `span_guid`, which [looks like a Root span][2]. Those assumptions may be wrong or unrelated, due my lack of understanding on how this should work internally as I'm not too familiar with this codebase or related contracts.

Please let me know how best I can support here to fix this use case. Thank you! 🙇 

**Environment**
* Elixir & Erlang version: 1.9.4, Erlang OTP 22
* Agent version: Elixir 1.17.1; Browser 1167; Ruby 6.10.0.364.

[1]: https://github.com/newrelic/rpm/blob/f42f233c02c3ad85ed4a8b41ea9bf2a664103d26/lib/new_relic/agent/transaction/distributed_tracing.rb#L160-L162
[2]: https://docs.newrelic.com/docs/understand-dependencies/distributed-tracing/trace-api/report-new-relic-format-traces-trace-api#recommended-attributes

**Comment by binaryseed** [comment]
Hello there! thanks for the report..

Could you send me the browser payload that you reference? My email is vince at...

**Comment by binaryseed** [comment]
Fixed in the latest 1.18 RC release!
